app/src/main/java/cm/aptoide/pt/AptoideApplication.java

```diff
@@ -918,5 +920,9 @@ public AdultContentAnalytics getAdultContentAnalytics() {
   public SettingsManager getSettingsManager() {
     return settingsManager;
   }
+
+  public AptoideDatabase getAptoideDatabase() {
+    return aptoideDatabase;
+  }
```

Comment:
I left it this way because this is how it was getting the database before (Realm one). Also, I figured it is better to dirty the Application (which is already dirty) than the AppviewFragment